lib/storage/acl.js

```diff
@@ -67,17 +69,138 @@ var util = require('../common/util.js');
  * @alias module:storage/acl
  */
 function Acl(options) {
+  AclRoleAccessorMethods.call(this);
+
   this.makeReq = options.makeReq;
   this.pathPrefix = options.pathPrefix;
 }
 
 /**
+ * A convenience object of methods to add or delete ACL permissions from an
+ * entity.
+ *
+ * The supported methods include:
+ *
+ *   - `myFile.acl.owners.addAllAuthenticatedUsers`
+ *   - `myFile.acl.owners.deleteAllAuthenticatedUsers`
+ *   - `myFile.acl.owners.addAllUsers`
+ *   - `myFile.acl.owners.deleteAllUsers`
+ *   - `myFile.acl.owners.addDomain`
+ *   - `myFile.acl.owners.deleteDomain`
+ *   - `myFile.acl.owners.addGroup`
+ *   - `myFile.acl.owners.deleteGroup`
+ *   - `myFile.acl.owners.addProject`
+ *   - `myFile.acl.owners.deleteProject`
+ *   - `myFile.acl.owners.addUser`
+ *   - `myFile.acl.owners.deleteUser`
+ *
+ * @alias acl.owners
+ *
+ * @return {object}
+ *
+ * @example
+ * //-
+ * // Add a user as an owner of a file.
+ * //-
+ * myFile.acl.owners.addUser('email@example.com', function(err, aclObject) {});
+ *
+ * //-
+ * // For reference, the above command is the same as running the following.
+ * //-
+ * myFile.acl.add({
+ *   entity: 'user-email@example.com',
+ *   role: storage.acl.OWNER_ROLE
+ * }, function(err, aclObject) {});
+ */
+Acl.prototype.owners = {};
+
+/**
+ * A convenience object of methods to add or delete ACL permissions from an
+ * entity.
+ *
+ * The supported methods include:
+ *
+ *   - `myFile.acl.readers.addAllAuthenticatedUsers`
+ *   - `myFile.acl.readers.deleteAllAuthenticatedUsers`
+ *   - `myFile.acl.readers.addAllUsers`
+ *   - `myFile.acl.readers.deleteAllUsers`
+ *   - `myFile.acl.readers.addDomain`
+ *   - `myFile.acl.readers.deleteDomain`
+ *   - `myFile.acl.readers.addGroup`
+ *   - `myFile.acl.readers.deleteGroup`
+ *   - `myFile.acl.readers.addProject`
+ *   - `myFile.acl.readers.deleteProject`
+ *   - `myFile.acl.readers.addUser`
+ *   - `myFile.acl.readers.deleteUser`
+ *
+ * @alias acl.readers
+ *
+ * @return {object}
+ *
+ * @example
+ * //-
+ * // Add a user as an owner of a file.
+ * //-
+ * myFile.acl.readers.addUser('email@example.com', function(err, aclObject) {});
+ *
+ * //-
+ * // For reference, the above command is the same as running the following.
+ * //-
+ * myFile.acl.add({
+ *   entity: 'user-email@example.com',
+ *   role: storage.acl.READER_ROLE
+ * }, function(err, aclObject) {});
+ */
+Acl.prototype.readers = {};
+
+/**
+ * A convenience object of methods to add or delete ACL permissions from an
```

Comment:
And finally writer role. Example: "A convenience object of methods to add and delete **writer** ACL permissions from different entities."